src/rtfjs/renderer/RenderChp.ts

```diff
@@ -75,5 +75,15 @@ export class RenderChp {
             }
         }
         el.style.fontSize = Math.floor(this._chp.fontsize / 2) + "pt";
+
+        if (this._chp.supersubscript === 1) {
+            el.style.verticalAlign = "super"
+        } else if (this._chp.supersubscript === -1) {
+            el.style.verticalAlign = "sub"
+        }
+
+        if (this._chp.supersubscript === 1 || this._chp.supersubscript === -1) {
+            el.style.fontSize = Math.floor((this._chp.fontsize / 2) - 2) + "pt"
```

Comment:
According to the spec we should `shrinks point size according to font information`, I haven't found how that is supposed to work though. As the result here matches the Libreoffice output quite well, I'd leave it as is for now.